test/wt_int_test.cpp

```diff
@@ -77,12 +77,22 @@ TYPED_TEST(wt_int_test, constructor)
         ASSERT_TRUE(store_to_file(wt, temp_file));
     }
     {
+        #if SDSL_HAS_CEREAL
```

Comment:
Please remove the indention for the `if` in this file as you did in the other files.